cmd/awat/import.go

```diff
@@ -29,44 +39,71 @@ var importCmd = &cobra.Command{
 	},
 }
 
-var getImportCmd = &cobra.Command{
-	Use:   "get",
-	Short: "Fetch an import by ID from the AWAT to get its status",
+var getImportByTranslationCmd = &cobra.Command{
+	Use:   "translation",
+	Short: "Get Imports by the Translation ID to which they correlate",
 	RunE: func(cmd *cobra.Command, args []string) error {
-		imprt, _ := cmd.Flags().GetString(importID)
 		server, _ := cmd.Flags().GetString(serverFlag)
-		_ = model.NewClient(server)
-
-		installation, _ := cmd.Flags().GetString(installationID)
+		translation, _ := cmd.Flags().GetString(id)
 		awat := model.NewClient(server)
-		if (installation == "" && imprt == "") || (installation != "" && imprt != "") {
-			return errors.New("one and only one of translation-id or import-id must be specified")
+		if translation == "" {
+			return errors.New("Must provide an Translation ID to search by Translation ID")
+		}
+		statuses, err := awat.GetImportStatusesByTranslation(translation)
+		if err != nil {
+			return err
 		}
-		if installation != "" {
-			status, err := awat.GetImportStatusesByInstallation(installation)
-			if len(status) > 0 {
-				_ = printJSON(status)
-			}
+		if len(statuses) > 0 {
+			_ = printJSON(statuses)
+			return nil
+		}
+		fmt.Println("No Imports found.")
+		return nil
+	},
+}
+
+var getImportByIDCmd = &cobra.Command{
+	Use:   "import",
+	Short: "Get an Import by its ID",
+	RunE: func(cmd *cobra.Command, args []string) error {
+		imprt, _ := cmd.Flags().GetString(id)
```

Comment:
nit: trying to save a letter here? :)

Reply:
no, `import` is a reserved word @gabrieljackson

Reply:
I also wouldn't name a variable `func` :stuck_out_tongue: